### ISSUE TITLE
Not build from Dockerfile with insecure registry

### ISSUE THREAD

**Issue by DmitriyMurzin** [genuinetools/img#126]
IMG not build from Dockerfile with insecure registry in section FROM

Dockerfile:
`FROM http://registry.local.com:5000/glassfish-app:88`

Error:
`failed to solve: invalid reference format`

 or 

Dockerfile:
`FROM registry.local.com:5000/glassfish-app:88`

Error:
```
 => ERROR docker-image://registry.local.com:5000/glassfish-app:88 
failed to solve: failed to do request: Head https://registry.local.com:5000/v2/glassfish-app/manifests/88: http: server gave HTTP response to HTTPS client
```

**Comment by jessfraz** [comment]
Can you not add self signed certs, we have info in the docs :) insecure is v bad

**Comment by DmitriyMurzin** [comment]
Im love insecure local registry!((

**Comment by ekristen** [comment]
@jessfraz I would love to see `--insecure-registry` flags on build and pull too! It's especially helpful for local development and testing when I don't want to put nginx or something else in front of the registry to serve https.

**Comment by Constantin07** [comment]
I agree, in local development/testing generating self signed certs is an overhead.
